doc/source/conf.py

```diff
@@ -62,6 +62,7 @@
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 sys.path.insert(0, os.path.abspath("../../python/"))
+sys.path.insert(0, os.path.abspath("../../python/ray/thirdparty_files"))
```

Comment:
Can you add a comment explaining what this is for? In particular, that it is to get psutil?

Another alternative is to just put `psutil` into requirements-doc.txt